src/TagSets.csv

```diff
@@ -24,7 +24,7 @@ Equipment>HVAC,Fan,"Any device with two or more blades or vanes attached to a ro
 Equipment>HVAC,Filter,Device to remove gases from a mixture of gases,ASHRAE Dictionary,,,,,,,,,,,,,,,
 Equipment>HVAC,Fume Hood,"A fume-collection device mounted over a work space, table, or shelf and serving to conduct unwanted gases away from the area enclosed.",ASHRAE Dictionary,,,,,,,,,,,,,,,
 Equipment>HVAC,Pump,"Machine for imparting energy to a fluid, causing it to do work, drawing a fluid into itself through an entrance port, and forcing the fluid out through an exhaust port.",ASHRAE Dictionary,,,,,,,,,,,,,,,
-Equipment>HVAC,Space Heater,"A heater used to warm the air in an enclosed area, such as a room or office","space heater. (n.d.) Collins English Dictionary � Complete and Unabridged, 12th Edition 2014. (1991, 1994, 1998, 2000, 2003, 2006, 2007, 2009, 2011, 2014). Retrieved February 20 2017 from http://www.thefreedictionary.com/space+heater",,,,,,,O,X,Jason,,,,,,
+Equipment>HVAC,Space Heater,"A heater used to warm the air in an enclosed area, such as a room or office","space heater. (n.d.) Collins English Dictionary  Complete and Unabridged, 12th Edition 2014. (1991, 1994, 1998, 2000, 2003, 2006, 2007, 2009, 2011, 2014). Retrieved February 20 2017 from http://www.thefreedictionary.com/space+heater",,,,,,,O,X,Jason,,,,,,
```

Comment:
These changes are not related to changing the Brick URI, so they should go under a separate pull request